src/components/mdButton/mdButton.vue

```diff
@@ -14,7 +14,7 @@
       let options = {
         staticClass: 'md-button',
         attrs: {
-          type: hasLink || 'button',
+          type: hasLink || this.type || 'button',
```

Comment:
I merged another PR that fixed this issue. So, after you correct this small thing, I can merge this one.